### DISCUSSION TITLE
Is there a way to configure a "global" theme that is automatically used everywhere?

### DISCUSSION THREAD

**Discussion by czottmann** [gonzalezreal/swift-markdown-ui#240] [Q&A] [answered]
I've created a basic theme, works fine:

```swift
public extension Theme {
  static let basicEnhanced = Theme()
```

To use it, I need to apply the theme every time I call the view:

```swift
Markdown("some **text**).markdownTheme(.basicEnhanced)
```

Is it possible to set automatically set a theme or would I need to create a custom view/wrapper to achieve that?

Thanks for an excellent package, Guille!

**Comment by gonzalezreal** [comment, marked as answer]
Hi @czottmann,

The `markdownTheme` modifier adds the theme to the environment of a view tree. So, if you use it in the app's root view, it will propagate to every child `Markdown` view, no matter how deep in the view hierarchy it is.

**Reply by czottmann** [reply]
Damn, that's good. Thank you!
